src/App.tsx

```diff
@@ -1,14 +1,57 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState, useMemo } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { Todo } from './types/Todo';
+
+import { getTodos } from './api';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const [allTodos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodo, selectTodo] = useState<Todo | null>(null);
+  const [filterBy, setFilterBy] = useState('all');
+  const [query, setQuery] = useState('');
+  const [isLoading, setLoading] = useState(true);
+
+  function toLowerCompare(str: string, part: string): boolean {
+    return str
+      .toLowerCase()
+      .includes(part.toLowerCase());
+  }
+
+  const visibleTodos = useMemo(() => {
+    switch (filterBy) {
+      case 'all':
+        return allTodos;
```

Comment:
Are you sure that you redeployed? Your filter works on demo link, but here you don’t filter by query when filter type is all 🤔